### PR TITLE
[flutter_tts] Always send valid values instead of Error

### PR DESCRIPTION
**Pull request by bbrto21**
* The implementation of this plugin expects to return a valid value such as 1,0.
So I mimicked other platform implementations.

Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

**Comment by bbrto21** [comment]
Fix #149

**Comment by HakkyuKim** [comment]
Was somewhat unsure about doing this fix, but if it's already done I'm OK with it! How about leaving a comment about the intentional API misuse in the source file? So that other engineers won't be confused about using `Success(flutter::EncodableValue(0))` to indicate errors.

**Comment by bbrto21** [comment]
@HakkyuKim I added some comments in source file, please review again~

File: packages/flutter_tts/CHANGELOG.md [hunk 1]
```diff
@@ -4,4 +4,9 @@
...
 ## 1.0.1
 
-* Stabilize a speaking states management.
\ No newline at end of file
+* Stabilize a speaking states management.
+
+## 1.1.0
+
+* All APIs now return valid values.(0 or 1, empty list, etc.)
+    * Now the native implementation doesn't throw an error to dart
```
Comment (swift-kim) [inline packages/flutter_tts/CHANGELOG.md:12]: Please refrain from using nested lists in the CHANGELOG file when you don't need to.

**Review by swift-kim — APPROVED** [review]

**Review by HakkyuKim — APPROVED** [review]
LGTM

**Pull request merged**